Localhost.postman_environment.json

```diff
@@ -0,0 +1,19 @@
+{
+	"id": "9ac49edf-797e-4719-8a04-680022ecd4fc",
+	"name": "Localhost",
+	"values": [
+		{
```

Comment:
indenting by 6 spaces?

Reply:
This file is an export from Postman, no control over their logic.